packages/coinstac-client-core/src/model-service.js

```diff
@@ -81,7 +84,7 @@ class ModelService {
   /**
    * @abstract
    */
-  modelServiceHooks() {
+  static modelServiceHooks() {
```

Comment:
You want me to pull it out then? It was one of the things reported by the linter: http://eslint.org/docs/rules/class-methods-use-this